pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -118,6 +118,9 @@ func (l *ScrapeConfigList) DeepCopyObject() runtime.Object {
 // ScrapeConfigSpec is a specification of the desired configuration for a scrape configuration.
 // +k8s:openapi-gen=true
 type ScrapeConfigSpec struct {
+	// The job name assigned to scraped metrics by default.
+	// +optional
+	JobName *string `json:"jobName,omitempty"`
```

Comment:
Let's add a validation that the string is at least 1 char if defined (e.g. empty isn't allowed).

Reply:
Added // +kubebuilder:validation:MinLength=1